lib/core/presentation/pages/collaborator/collaborator_discover/widgets/collaborator_discover_spotline_events_section.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class CollaboratorDiscoverSpotlineEventsSection extends StatelessWidget {
+  const CollaboratorDiscoverSpotlineEventsSection({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final t = Translations.of(context);
+    final colorScheme = Theme.of(context).colorScheme;
+    final events = [
+      Event(
+        title: "Living room gig gigachad nomad",
+        start: DateTime.now(),
+      ),
+      Event(
+        title: "Early morning yoga",
+        start: DateTime.now(),
+      ),
+      Event(
+        title: "Secret room",
+        start: DateTime.now(),
+      ),
+    ];
+    if (events.isEmpty) {
+      return const SliverToBoxAdapter(child: SizedBox.shrink());
+    }
+    return SliverToBoxAdapter(
+      child: Container(
+        padding: EdgeInsets.only(
+          left: Spacing.smMedium,
+          top: Spacing.smMedium,
+          bottom: Spacing.smMedium,
+        ),
+        clipBehavior: Clip.antiAlias,
+        decoration: ShapeDecoration(
+          color: LemonColor.white06,
+          shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.circular(LemonRadius.normal),
+          ),
+        ),
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          mainAxisAlignment: MainAxisAlignment.center,
+          crossAxisAlignment: CrossAxisAlignment.end,
+          children: [
+            SizedBox(
+              width: double.infinity,
+              child: Row(
+                mainAxisSize: MainAxisSize.min,
+                mainAxisAlignment: MainAxisAlignment.start,
+                crossAxisAlignment: CrossAxisAlignment.center,
+                children: [
+                  Container(
+                    width: 18,
+                    height: 18,
+                    clipBehavior: Clip.antiAlias,
+                    decoration: const BoxDecoration(),
+                    child: ThemeSvgIcon(
+                      color: LemonColor.paleViolet,
+                      builder: (colorFilter) => Assets.icons.icHouseParty.svg(
+                        colorFilter: colorFilter,
+                      ),
+                    ),
+                  ),
+                  SizedBox(width: Spacing.smMedium / 2),
+                  Expanded(
+                    child: SizedBox(
+                      child: Text(
+                        t.collaborator.spotlightEvents,
+                        style: Typo.medium.copyWith(
+                          color: colorScheme.onPrimary,
+                          fontWeight: FontWeight.w600,
+                        ),
+                      ),
+                    ),
+                  ),
+                ],
+              ),
+            ),
+            SizedBox(height: Spacing.smMedium),
+            SizedBox(
+              height: 128.w,
+              child: ListView.separated(
+                separatorBuilder: (context, index) {
```

Comment:
add a left padding to ListView, instead of left padding on above container, it will make the list look better (refer to the design)
<img width="512" alt="Screenshot 2024-04-29 at 09 11 43" src="https://github.com/lemonadesocial/lemonade-flutter/assets/28992487/d7536512-2c6a-4133-a796-4f54b2457c3b">